components/net/src/heartbeat.rs

```diff
@@ -0,0 +1,68 @@
+use error::Result;
+use server::ZMQ_CONTEXT;
+use zmq;
+
+/// In-memory zmq address for HeartbeatMgr
+const INPROC_ADDR: &'static str = "inproc://heartbeat";
+/// Protocol message to notify the HeartbeatMgr to begin pulsing
+const CMD_PULSE: &'static str = "R";
+/// Protocol message to notify the HeartbeatMgr to pause pulsing
+const CMD_PAUSE: &'static str = "P";
+
+#[derive(PartialEq)]
+enum PulseState {
```

Comment:
Should PulseState also get removed from builder-worker/src/heartbeat.hs?  Looks like there are duplicates.